curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646add141ce9f901b37f6d2b.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 646add141ce9f901b37f6d2b
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+You will create a `main` element inside the `body` element that will contain all the elements of the project.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed